x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/elastic_endpoint_security_behavior_protection.json

```diff
@@ -1,68 +0,0 @@
-{
-  "author": [
-    "Elastic"
-  ],
-  "description": "Generates a detection alert each time an Elastic Endpoint Security alert is received for Behavior Protection alerts. Enabling this rule allows you to immediately begin investigating your Endpoint alerts for Behavior Protection.",
-  "enabled": true,
-  "exceptions_list": [
-    {
-      "id": "endpoint_list",
-      "list_id": "endpoint_list",
-      "namespace_type": "agnostic",
-      "type": "endpoint"
-    }
-  ],
-  "from": "now-10m",
-  "index": [
-    "logs-endpoint.alerts-*"
-  ],
-  "language": "kuery",
-  "license": "Elastic License v2",
-  "max_signals": 10000,
-  "name": "Endpoint Security Behavior Protection",
-  "query": "event.kind:alert and event.module:(endpoint and not endgame) and event.code: behavior\n",
```

Comment:
we removed this rule from the detection-rules repository, same reason as the other